omp/CMakeLists.txt

```
@@ -28,7 +28,12 @@ target_sources(ginkgo_omp
 
 ginkgo_compile_features(ginkgo_omp)
 target_link_libraries(ginkgo_omp PRIVATE "${OpenMP_CXX_LIBRARIES}")
-target_compile_options(ginkgo_omp PRIVATE "${OpenMP_CXX_FLAGS}")
+target_include_directories(ginkgo_omp PRIVATE "${OpenMP_CXX_INCLUDE_DIRS}")
+if(APPLE)
```

Actually, if you are going to do this, you should do it for AppleClang only. See: https://cmake.org/cmake/help/v3.0/variable/CMAKE_LANG_COMPILER_ID.html#variable:CMAKE_%3CLANG%3E_COMPILER_ID

```suggestion
if(CMAKE_CXX_COMPILER_ID MATCHES "AppleClang")
```